test/functional/test_framework/messages.py

```diff
@@ -1160,6 +1160,26 @@ def serialize(self):
     def __repr__(self):
         return "msg_mempool()"
 
+
+class msg_notfound():
+    command = b"notfound"
+
+    def __init__(self, vec=None):
+        if vec is None:
+            self.vec = []
+        else:
+            self.vec = vec
+
+    def deserialize(self, f):
```

Comment:
Could use a better variable name than `f`? :-)

Reply:
This maintains consistency with existing methods in similar classes (and it's a one-line usage: not considerably different from, say, a list comprehension) so I think this is fine.

Reply:
Now I see that `f` is used consistently. Makes sense to follow that.